app/api/v2/managers/schedule_api_manager.py

```diff
@@ -0,0 +1,37 @@
+from marshmallow.schema import SchemaMeta
+from typing import Any
+
+from app.api.v2.managers.operation_api_manager import OperationApiManager
+from app.objects.c_operation import OperationSchema
+from app.utility.base_world import BaseWorld
+
+
+class ScheduleApiManager(OperationApiManager):
+    def __init__(self, services):
+        super().__init__(services)
+        self.services = services
+
+    def find_and_update_object(self, ram_key: str, data: dict, search: dict = None):
+        return super(OperationApiManager, self).find_and_update_object(ram_key, data, search)
+
+    def update_object(self, obj: Any, data: dict):
+        return super(OperationApiManager, self).update_object(obj, data)
+
+    def create_object_from_schema(self, schema: SchemaMeta, data: dict, access: BaseWorld.Access):
+        return super(OperationApiManager, self).create_object_from_schema(schema, data, access)
+
+    async def setup_operation(self, data: dict, access: BaseWorld.Access):
```

Comment:
This overlaps greatly with `setup_operation` from operation_api_manager.py. I'm thinking we can just call that function here - the original function does call `operation.set_start_details()` in order to set the operation start time - that line will need to be moved out of the function and into the caller functions (e.g. in `create_object_from_schema` for operation_api_manager.py). Then you can just re-use that function here, provided you do the state validation beforehand.